src/index.js

```diff
@@ -47,9 +50,19 @@ class LumigoPlugin {
 		).toLowerCase();
 	}
 
+	async afterDeployFunctionInitialize() {
+		await this.wrapFunctions([this.options.function]);
+	}
+
 	async afterPackageInitialize() {
-		// [{"handler":"handler.hello","events":[],"name":"aws-nodejs-dev-hello"}]
-		const { runtime, functions } = this.getFunctionsToWrap(this.serverless.service);
+		await this.wrapFunctions();
```

Comment:
Are you missing here `service.getAllFunctions()` ?

Reply:
No it is done in `wrapFunctions()`